zigpy/types/basic.py

```diff
@@ -81,6 +81,7 @@ class FixedIntType(int):
     _signed = None
     _bits = None
     _size = None  # Only for backwards compatibility, not set for smaller ints
+    _bigendian = None
```

Comment:
I think `_byteorder` and turning it into a string would be a better approach, it matches the `int.from_bytes` API and I think is a clear name.